syncany-lib/src/main/java/org/syncany/operations/up/Indexer.java

```diff
@@ -91,10 +89,18 @@
 
 	private LocalEventBus eventBus;
 
+
 	public Indexer(Config config, Deduper deduper) {
 		this.config = config;
 		this.deduper = deduper;
 		this.localDatabase = new SqlDatabase(config);
+		try {
+			localDatabase.getConnection().setReadOnly(true);
```

Comment:
try/catch block in a constructor with `e.printStackTrace()`? Shouldn't this be something like `SqlDatabase.createReadOnly(config);` or something?